libunwind/src/AddressSpace.hpp

```diff
@@ -408,6 +414,24 @@ inline uint64_t LocalAddressSpace::getRegister(pint_t addr) {
 #endif
 }
 
+#if defined(__CHERI_PURE_CAPABILITY__) && defined(_LIBUNWIND_CHERI_C18N_SUPPORT)
+extern "C" {
+/// Call into the RTLD to get a sealer capability. This sealer will be used to
+/// seal information in the unwinding context.
+uintptr_t _rtld_unw_getsealer(void);
```

Comment:
```suggestion
uintptr_t _rtld_unw_getsealer();
```
`(void)` and `()` are the same in C++ (and C23), and if they weren't you should've also used the former for the weak fallback.